### PR TITLE
Tests pending for issue #5

### PR DESCRIPTION
**Pull request by hashhar**
This is what I've done. It's under testing for edge cases. I'll report by 6:30 am GMT.

It seems from the changes in the `wakatime-core` that this is the only way. Since we do not explicitly say when we lost internet. The only errors are 102, 103, 401.

The way I have done it seems to introduce a lot of redundancy. I'll try refactoring it.

### CODE REVIEW

**Comment by hashhar** [comment]
This doesn't seem to work exactly as I'd want it. Needs further tweaking.

File: extension.ts [hunk 1]
```diff
@@ -127,12 +127,24 @@ export class WakaTime {
...
+                        this.statusBar.tooltip = 'It seems you are not connected to the internet.';
                         if (code == 102) {
+                            this.statusBar.text = '$(clock) WakaTime Error';
+                            this.statusBar.tooltip = 'Help -> Toggle Developer Tools for more details';
                             console.error('API Error (102); Check your ~/.wakatime.log file for more details.');
                         } else if (code == 103) {
+                            this.statusBar.text = '$(clock) WakaTime Error';
+                            this.statusBar.tooltip = 'Help -> Toggle Developer Tools for more details';
                             console.error('Config Parsing Error (103); Check your ~/.wakatime.log file for more details.');
+                        } else if (code == 401) {
```
Comment (alanhamlett) [inline extension.ts]: This should be `} else if (code == 104) {` which is `AUTH_ERROR` from [wakatime/constants.py](https://github.com/wakatime/wakatime/blob/master/wakatime/constants.py#L16).

**Comment by alanhamlett** [comment]
Fixed with 71f0f1a1230edb494a8fcc8e3eec2a5bdb6bda3f and released in v1.0.3.

Thank you @hashhar!
